specs/vep-1507-vdk-operations-ui/README.md

```diff
@@ -164,6 +166,215 @@ Consider at least the below topics but you do not need to cover those that are n
       * Is it logged?
   * What secrets are needed by the components? How are these secrets secured and attained?
 -->
+### User Journey
+
+#### Frontend components
```

Comment:
We discussed the menus with @sabadzhiev. We should put that `Explore` page behind a feature flag for the open source version. https://github.com/vmware/versatile-data-kit/issues/1751

Reply:
Great, looking forward to the structure that enumerates the use cases, and reflects on the Navigation we currently have 😊

In case we have a TODO, you can link it, along with the ticket, to the VEP as well. For example, a decision has been made to exclude Explore use cases, so we will add a feature flag in ticket #...